README.md

```diff
@@ -8,7 +8,7 @@ python setup.py test
 
 # How to package
 ```
-python setup.py sdist
+python setup.py sdist bdist_wheel
```

Comment:
It doesn't work for me:
```
GDSpb0659:simple-python-service dmyasnikov$ python setup.py sdist bdist_wheel
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/setuptools/dist.py:406: UserWarning: The version specified ('0.4-SNAPSHOT') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
  "details." % self.metadata.version
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'bdist_wheel'
```

Reply:
```pip install wheel```

Reply:
Fixed